portals/knowledge-management/src/Pages/ServiceMonitorStatus.vue

```diff
@@ -0,0 +1,211 @@
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Aiding App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    <https://www.canyongbs.com> or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+-->
+<script setup>
+    import { XMarkIcon } from '@heroicons/vue/24/outline';
+    import { computed, onMounted, ref } from 'vue';
+    import Breadcrumbs from '../Components/Breadcrumbs.vue';
+    import Pagination from '../Components/Pagination.vue';
+    import ServiceMonitorAlert from '../Components/ServiceMonitorAlert.vue';
+    import ServiceMonitorCard from '../Components/ServiceMonitorCard.vue';
+    import { consumer } from '../Services/Consumer.js';
+
+    const emit = defineEmits(['fetchNextPage', 'fetchPreviousPage', 'fetchPage', 'change-filter']);
+
+    const result = ref([]);
+    const { get } = consumer();
+    const loading = ref(true);
+
+    const currentPage = ref(1);
+    const nextPageUrl = ref(null);
+    const prevPageUrl = ref(null);
+    const lastPage = ref(null);
+    const totalArticles = ref(0);
+    const fromArticle = ref(0);
+    const toArticle = ref(0);
+
+    const props = defineProps({
+        apiUrl: {
+            type: String,
+            required: true,
+        },
+    });
+
+    const okTitle = 'All systems operational';
+    const okMessage =
+        'All systems are functioning seamlessly, with no disruptions or downtime reported. Every component, from critical infrastructure to auxiliary services is running at full capacity, ensuring optimal performance and reliability.';
+
+    const issueTitle = 'Some systems are experiencing issues';
+    const issueMessage = 'One or more services are currently experiencing disruptions or downtime.';
+
+    const hasIssues = computed(() =>
+        result.value.some(
+            (serviceMonitor) =>
+                serviceMonitor.latest_history !== null && serviceMonitor.latest_history.succeeded !== true,
```

Comment:
I don't think we should consider a service monitor not having any history as an issue.

It should probably count as there not being any problems. Since we have 24 hour checks this would mean that when someone added a new 24 hour check it would say there are problems until the check is first run later.